packages/itwinui-css/backstop/tests/slider.html

```diff
@@ -491,19 +664,17 @@ <h2>Default vertical</h2>
           </div>
           <span class="iui-slider-max">100</span>
         </div>
-
       </div>
 
       <div class="iui-input-container iui-disabled">
         <div class="iui-label">Disabled slider</div>
-
-        <div
-          class="iui-slider-component-container iui-slider-vertical iui-input-group iui-disabled"
-          style="height: 500px;"
-        >
+        <div class="iui-slider-container iui-slider-vertical iui-input-group iui-disabled">
```

Comment:
some of these are messed up (presumably you're aware of it)
<img width="509" alt="" src="https://github.com/iTwin/iTwinUI/assets/9084735/d7634129-d4a4-4226-86d6-e329ed45e976">

if we can avoid using `iui-input-group`, that might help. it was never designed for this purpose.